site/src/AppRouter.tsx

```diff
@@ -27,167 +29,173 @@ const WorkspacesPage = lazy(() => import("./pages/WorkspacesPage/WorkspacesPage"
 const CreateWorkspacePage = lazy(() => import("./pages/CreateWorkspacePage/CreateWorkspacePage"))
 const AuditPage = lazy(() => import("./pages/AuditPage/AuditPage"))
 
-export const AppRouter: FC = () => (
-  <Suspense fallback={<></>}>
-    <Routes>
-      <Route
-        index
-        element={
-          <RequireAuth>
-            <IndexPage />
-          </RequireAuth>
-        }
-      />
+export const AppRouter: FC = () => {
+  const xServices = useContext(XServiceContext)
+  const [authState] = useActor(xServices.authXService)
+  const { permissions } = authState.context
```

Comment:
This is probably a good place to use `useSelector` instead of `useActor` so that we don't get re-renders for anything except permissions. In fact, I think my plan was to do that within a wrapper component so that it only gets accessed on pages that need it (I'll add a link in a sec)

Reply:
I'm realizing now that I was thinking about licensing instead of role-based permissions, but maybe it's still a good idea? Up to you!

Reply:
That is a good idea! I will play around with the wrapper component when I get to licensing, and make sure permissions go in there, too, if possible.